src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -595,39 +594,335 @@ public override bool IsTupleType
             }
         }
 
-        internal sealed override ObsoleteAttributeData ObsoleteAttributeData
+        public override bool IsImplicitlyDeclared
         {
             get
             {
-                return null;
+                return false;
+            }
+        }
+
+        public override ImmutableArray<TypeSymbol> TupleElementTypes
+        {
+            get
+            {
+                return _elementTypes;
+            }
+        }
+
+        public override ImmutableArray<string> TupleElementNames
+        {
+            get
+            {
+                return _elementNames;
             }
         }
 
         public override ImmutableArray<Symbol> GetMembers()
```

Comment:
This is a standard method that every type symbol has to implement. Not a new API

---
In reply to: [61819121](https://github.com/dotnet/roslyn/pull/11006#discussion_r61819121) [](ancestors = 61819121)